content/features/05-graph-types.adoc

```diff
@@ -0,0 +1,11 @@
+---
+title: "Graph: Types"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+image: features/graph-types.png
+subTitle: "Choosing what kind of graph to display"
+weight: 01
+---
+
+The **workload** graph will provide a graph showing the interactions between workloads. You can also choose to group workloads by the app label in the **app** graph, or group workloads by the app and version label in the **versioned app** graph. For a graph that only shows services, use the **service** graph type.
```

Comment:
"the workload graph will provide a graph" sounds a bit strange. 
I suggest "The workload graph focuses on visualizing the communication between workloads"

Reply:
ok